src/components/FormComponents.tsx

```diff
@@ -1,9 +1,32 @@
-import React from 'react';
-import { useField } from 'formik';
+import React, { ReactNode } from 'react';
+import {
+  Formik,
+  FormikProps,
+  useField,
+  FormikConfig,
+  FormikValues,
+} from 'formik';
 import classnames from 'classnames';
 import { UserInput } from '../types/UserInput';
 import { numberInputRegexp } from '../lib/utils';
 
+export type FormWrapperProps<FormikInput> = FormikConfig<FormikInput> & {
```

Comment:
Nieco mi vravi ze tu by sa niekde mohlo dat pouzit nieco ako `FormWrapper: typeof Formik` ale nie som si isty ci sa to naozaj da

Reply:
To som skusal, ale vracia mi to viac chybu na `children` a neviem to spravit potom genericke...